ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/ContentTest.kt

```diff
@@ -184,6 +184,7 @@ class ContentTest : ClientLoader(5 * 60) {
                     }
                 }
                 append("hello", 5)
+                append("world", true)
```

Comment:
This test is actually annotated with `@Ignore`, I did a local run with annotation removed though.